iterations_runner.lua

```diff
@@ -40,6 +40,34 @@ local ffi = require("ffi")
 local json = require("json_nojit")
 local jitstats
 
+function add_luapackage_path(basepath, highpriority)
+    assert(type(basepath) == "string" and #basepath > 0)
+
+    if string.sub(basepath, -1) == "/" then
```

Comment:
This isn't fully safe: for example `///blah` is a valid basepath. Is there a Lua library that manipulates paths that we can use? Getting this stuff right, and portable, is harder than it seems...

Reply:
this is why i asked for krun to provide it

Reply:
How do we know what comes out of Krun is fully normalised? Let's not manipulate paths as strings -- it's just a recipe for disaster.

Reply:
I mean for krun to pass the script directory path as a argument to the runner

Reply:
I understand that. But you can't guarantee it's normalised. So let's please use a library to fix this.